src/stackdriver-nozzle/nozzle/nozzle.go

```
@@ -33,7 +36,12 @@ func (n *Nozzle) HandleEvent(envelope *events.Envelope) error {
 		return nil
 	} else {
 		metrics := n.Serializer.GetMetrics(envelope)
-		return n.postMetrics(metrics)
+		err := n.postMetrics(metrics)
+		if err != nil {
+			n.Logger.Error("metricError", err)
+		}
+		//TODO: should we return the error or only log it?
```

i believe convention is to append context to an error and let it bubble up:
```go
if err != nil {
   return fmt.Errorf("metricError: %v", err)
}
```